tests/test_sagemaker/test_sagemaker.py

```diff
@@ -0,0 +1,79 @@
+"""Unit tests for sagemaker-supported APIs."""
+import boto3
+
+from moto import mock_sagemaker
+
+# See our Development Tips on writing tests for hints on how to write good tests:
+# http://docs.getmoto.org/en/latest/docs/contributing/development_tips/tests.html
+
+
+@mock_sagemaker
+def test_list_model_packages():
```

Comment:
A large part of the logic for this method is how it filters on various attributes, so that should be tested as well.

Reply:
https://github.com/getmoto/moto/pull/6454/commits/edf2765de0aba3910d01d56642191cc286ca5d66